src/lib.rs

```diff
@@ -718,7 +718,7 @@ impl Config {
                     parallel_flags = Some(format!("-j{}", s));
                 }
                 Some(ref g) if g.contains("Visual Studio") => {
-                    parallel_flags = Some(format!("/m:{}", s));
+                    parallel_flags = Some(format!("/MP{}", s));
```

Comment:
To confirm, you've seen this have parallelized effects locally for yourself? I'm not sure how `cmake` invokes `cl.exe` but it looks like this only works if `cl.exe` is fed multiple files at once? I'm also not sure what the previous `/m` option was actually...

Reply:
Yes. I haven't come across the m flag before personally.